tasks/DisableCpuFreqScaling.rb

```diff
@@ -0,0 +1,16 @@
+class DisableCpuFreqScaling
+  include Genesis::Framework::Task
+
+  init do
+    log "Make sure acpi-cpufreq is enabled" 
+    run_cmd('modprobe acpi-cpufreq')
```

Comment:
Yea fair enough. With those services and others, you need to do a bit more configuration to set the profile. Most default to a balanced or power saving mode. Best to stop all the cpuspeed services and manually set them to performance I suppose